CppHeaderParser/CppHeaderParser.py

```diff
@@ -2453,15 +2453,15 @@ def _evaluate_class_stack(self):
             del self.nameStack[0]
 
         if len(self.nameStack) == 1:
-            if self.nameStack[0] == "struct":
+            if "struct" in self.nameStack:
```

Comment:
This doesn't make sense. If len(self.nameStack) is 1, then there should only be a single element?

Reply:
This was in anticipation for fixing the "parser does not parse nested classes with preprocessor keywords" bug, which needs more work to get fixed.
However, this is more general and is therefore a better practice imo ?

Reply:
I know that CppHeaderParser does a lot of `'name' in namestack`, but I'm trying to get away from that. I'd rather something parse out a separable piece, then check *those* elements instead.

Anyways, this doesn't add any value as is and will just confuse future readers until someone adds the feature you're talking about, so let's remove it for now.